src/formatting.rs

```diff
@@ -705,7 +767,25 @@ fn make_parse_sess(source_map: Rc<SourceMap>, config: &Config) -> ParseSess {
         } else {
             ColorConfig::Never
         };
-        Handler::with_tty_emitter(color_cfg, true, None, Some(source_map.clone()))
+
+        let emitter_writer =
+            EmitterWriter::stderr(color_cfg, Some(source_map.clone()), false, false, None);
+        let emitter = Box::new(SilentOnIgnoredFilesEmitter {
+            has_non_ignorable_parser_errors: false,
+            can_reset: false,
+            ignore_path_set: ignore_path_set,
+            source_map: source_map.clone(),
+            emitter: emitter_writer,
+            parser_error_resetter,
+        });
+        Handler::with_emitter_and_flags(
```

Comment:
Nit: could we use `Handler::with_emitter` here as well?

Reply:
Yeah I think we probably can. I think I used `with_emitter_and_flags` at one point to see if there were any helpful flags I could use and forgot to change it back 😄